package.json

```diff
@@ -89,6 +89,14 @@
     "qunit-dom": "1.6.0",
     "sass": "1.32.5"
   },
+  "//": {
+    "uikit": "Higher versions break the build. https://github.com/uikit/uikit/issues/4403",
+    "@babel/parser": "For some reason 7.12.13 fails to transpile TypeScript sources. https://github.com/babel/babel/issues/12745"
```

Comment:
What about 7.12.12?

Reply:
Couldn't install it. Eventhough there is a tag on GitHub Yarn couldn't find a package and showed me the list of other versions.

Reply:
:rofl: Indeed, there's no such version on npm so nvm.